include/mbedtls/ssl_internal.h

```diff
@@ -1229,4 +1229,184 @@ void mbedtls_ssl_buffering_free( mbedtls_ssl_context *ssl );
 void mbedtls_ssl_flight_free( mbedtls_ssl_flight_item *flight );
 #endif /* MBEDTLS_SSL_PROTO_DTLS */
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
```

Comment:
In the development branch, we now tend to declare library-internal functions in header files under `library`. We didn't go and move existing "morally internal but technically public" things yet, this is planned 3.0, but for new functions we favor using real private headers.

I thing this could go in `library/ssl_tls13_keys.h`.

Reply:
Will do :+1: